src/Symfony/Component/String/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.2
+---
+
+ * Add `TruncateMode::WordBefore` mode to truncate method
```

Comment:
It's really the full `TruncateMode` enum. Maybe we should mention that how to map the bools to the new modes?

Reply:
You are right :+1: Fixed